lib/archive.ts

```diff
@@ -151,13 +151,20 @@ async function getMavenPackageInfo(
         if (err) {
           reject(err);
         }
-        if (!res || !res.response || res.response.docs.length === 0) {
+        if (!res || !res.response) {
           reject(
             new Error(
-              `No package found querying '${MAVEN_SEARCH_URL}' for sha1 hash '${sha1}'.`,
+              `Unexpected result querying '${MAVEN_SEARCH_URL}' for sha1 hash '${sha1}'.`,
             ),
           );
         }
+        if (res.response.docs.length === 0) {
+          resolve({
+            g: 'unknown',
+            a: `${targetPath}:${sha1}`,
+            v: 'unknown',
```

Comment:
here we might have a version or group at this point but we decide to mark both as `unknown` to flag that this is where we cut the transitive path because we couldn't resolve it, right?

Reply:
we don't have either, marking version as 'unknown' makes the UI alert the user, marking group as unknown allows the user to search for them